src/test/java/net/kaczmarzyk/spring/data/jpa/domain/NotEqualIgnoreCaseTest.java

```diff
@@ -51,6 +64,13 @@ public void equalsAndHashCodeContract() {
 				.verify();
 	}
 
+	@Test
+	public void toStringVerifier() {
+		ToStringVerifier.forClass(NotEqualIgnoreCase.class)
+				.withIgnoredFields("path", "queryContext")
```

Comment:
why path is ignored? (maybe it should be, but at first glance looks like a good candidate for toString?)